server/src/test/scala/services/graphstages/ArrivalsGraphStageLiveBaseArrivalsSpec.scala

```diff
@@ -105,13 +106,176 @@ class ArrivalsGraphStageLiveBaseArrivalsSpec extends CrunchTestLike with AfterEa
+
+    probe.fishForMessage(2 seconds) {
+      case ArrivalsDiff(toUpdate, _) =>
+        toUpdate.exists {
+          case (_, a) =>
+            a.ScheduledDeparture.get == scheduledDeparture &&
+            a.Scheduled == scheduled.millisSinceEpoch
+        }
+    }
+
+    success
+  }
+
+  "Given a forecast arrival with a scheduled arrival time and a live arrival with in 60 minutes scheduled arrival time" +
+    "Then they should not merged and scheduled Departure not present" >> {
+    val probe = TestProbe("arrivals")
+    val (forecastBaseArrivalsSource, _, _, liveSource) = TestableArrivalsGraphStage(probe, buildArrivalsGraphStage).run
+
+    val scheduledDeparture = scheduled.addHours(-4).millisSinceEpoch
+    forecastBaseArrivalsSource.offer(List(arrival(scheduledDate = scheduled.addMinutes(59).millisSinceEpoch, scheduledDepartureDate = Some(scheduledDeparture))))
+
+    probe.fishForMessage(2 seconds) {
+      case ArrivalsDiff(toUpdate, _) =>
+        toUpdate.exists {
+          case (_, _) => true
+        }
+    }
+
+    liveSource.offer(List(arrival()))
+
+    probe.fishForMessage(2 seconds) {
+      case ArrivalsDiff(toUpdate, _) =>
+        toUpdate.exists {
+          case (_, a) => a.ScheduledDeparture == None
+        }
+    }
+
+    success
+  }
+
+
+  "Given a forecast base arrival with a scheduled arrival time and a live  base arrival with in 60 minutes scheduled arrival time and forecast arrival with scheduled same as forecast base" +
+    "Then they should  merged and scheduled Departure present and actual pax count should exists" >> {
+    val probe = TestProbe("arrivals")
+    val (forecastBaseArrivalsSource, forecastArrivalSource, liveBaseSource, _) = TestableArrivalsGraphStage(probe, buildArrivalsGraphStage).run
+
+    val scheduledDeparture = scheduled.addHours(-4).millisSinceEpoch
+
+    forecastBaseArrivalsSource.offer(List(arrival()))
+
+    probe.fishForMessage(2 seconds) {
+      case ArrivalsDiff(toUpdate, _) =>
+        toUpdate.exists {
+          case (_, _) => true
+        }
+    }
+
+    liveBaseSource.offer(List(arrival(scheduledDate = scheduled.addMinutes(59).millisSinceEpoch, scheduledDepartureDate = Some(scheduledDeparture))))
+
+    probe.fishForMessage(2 seconds) {
+      case ArrivalsDiff(toUpdate, _) =>
+        toUpdate.exists {
+          case (_, a) => a.ScheduledDeparture.get == scheduledDeparture
+            a.Scheduled == scheduled.millisSinceEpoch
+        }
+    }
+
+    forecastArrivalSource.offer(List(arrival(actPax=Some(90))))
+
+    probe.fishForMessage(2 seconds) {
+      case ArrivalsDiff(toUpdate, _) =>
+        toUpdate.exists {
+          case (_, a) => a.ScheduledDeparture.get == scheduledDeparture && a.ActPax.get == 90
+        }
+    }
+
+    success
+  }
+
+  "Given a live arrival with a scheduled arrival time and a live base arrival with in 60 minutes scheduled arrival time and forecast arrival with scheduled same as forecast base" +
```

Comment:
Is this the intended logic? I'm not sure why they wouldn't get merged if cirium is within 60 minutes of the live arrival

Reply:
I just forgot change the text of test